File: Python/QUESTÃO 9.py
userAge  = int(input('To find out if you can retire, enter your age: '))
jobTime = int(input('Now, enter how many years of service you have: '))

if userAge >= 65:
    print('Congratulations, you can retire!!')
elif jobTime >= 30:
    print('Congratulations, You can retire!!')
elif userAge >=60 and jobTime >= 25:
    print('Congratulations, You can retire!!.')
else:
    print('You cannot retire yet.')

File: JavaScript/QUESTÃO 9.js
let userAge = Number(prompt("To find out if you can retire, enter your age: "))
let jobTime = Number(prompt("Now, enter how many years of service you have:"))

if (isNaN(userAge) || userAge <0 || isNaN(jobTime) || jobTime <0) {
    console.log("Invalid Input!!. Please enter you age and you work time.")
} else if (userAge >= 65 || jobTime >= 30 || (userAge === 60 && jobTime === 25)) {
    console.log("Congratulations, you can retire!!")
} else {
    console.log("You cannot retire yet.")
}